docs/book/advanced-guide/stacks/custom-flavors.md

```diff
@@ -248,6 +248,11 @@ used, check out the [Managing Stack Component States](./stack-state-management.m
 section for more details.
 {% endhint %}
 
+Check out [this short (< 3 minutes) video](https://www.youtube.com/watch?v=CQRVSKbBjtQ) on how to quickly get some more
+information about the registered flavors available to you:
+
+[![Describing Flavors 🌮 Makes Your Life Easier](../assets/flavors-video-snapshot.png)](https://www.youtube.com/watch?v=CQRVSKbBjtQ)
```

Comment:
There is a much easier way to do this using the special normalized syntax Gitbook has:

{% embed url="https://www.youtube.com/watch?v=CQRVSKbBjtQ" %} Describe MLOps Stack Component Flavors {% endembed %}



See [here](https://github.com/GitbookIO/git-sync-normalization/blob/e52d36744adcbe396dc3ab5126ee27725be498ac/embeds.md) as a reference